lib/bindConnectors/bindDynamicOutput.js

```diff
@@ -0,0 +1,31 @@
+var _                = require('lodash');
+var when             = require('when');
+var GenerateSchema = require('generate-schema');
+
+module.exports = function (message, options) {
+
+    return function (event) {
+
+        return when.promise(function (resolve, reject) {
+
+            when(message.dynamicOutput(event.body.input))
```

Comment:
Request & response messages are different in the format that comes in. The connector receives it like this: https://github.com/trayio/connector-spec/wiki/Lambda-connectors